versioned_docs/version-2.0/how_to_guides/datasets/index_datasets_for_dynamic_few_shot_example_selection.mdx

```diff
@@ -11,32 +11,57 @@ import {
 # Dynamic few shot example selection
 
 :::note
-This feature is currently in closed beta. Please sign up [here](https://forms.gle/in9R6t9HNSYMBt7P7) for access
+This feature is in open beta. It is only available to paid team plans. Please reach out to support@langchain.dev
+if you have questions about enablement.
 :::
 
 Configure your datasets so that you can search for few shot examples based on an incoming request.
 
 ## Pre-conditions
 
-1. Your dataset must have the KV store data type (we do not currently support chat model or LLM type datasets)
+1. Your dataset must use the KV store data type (we do not currently support chat model or LLM type datasets)
 2. You must have an input schema defined for your dataset. See our docs on setting up schema validation [in our UI](./manage_datasets_in_application#dataset-schema-validation) for details.
-3. You must be enabled for the closed beta
+3. You must be on a paid team plan (e.g. Plus plan)
 4. You must be on LangSmith cloud
 
-## Index your dataset to be searched
+## Index your dataset for few shot search
 
-On the datasets UI, click the `Few-shot Index` on the top right corner and hit `Start Sync`.
+Navigate to the datasets UI, and click the new `Few-Shot search` tab. Hit the `Start sync` button, which will
+create a new index on your dataset to make it searchable.
 
-![](../static/few-shot-index.png)
+![](../static/few_shot_tab_unsynced.png)
 
-This process will start to index your data to be searchable in the background. A note will appear on the
-modal above that says if your index is up to date, and if not, what version of your dataset it last indexed.
+By default, we sync to the latest version of your dataset. That means when new examples are added to your dataset, they
+will automatically be added to your index. This process runs every few minutes, so there should be a very short delay for
+indexing new examples. You can see whether your index is up to date under `Few-shot index` on the lefthand side of the screen
+in the next section.
 
-All new data added to your dataset will automatically be indexed. You do not need to re-index when adding new data.
+## Test search quality in the few shot playground
 
-## Search your dataset for similar examples
+Now that you have turned on indexing for your dataset, you will see the new few shot playground.
 
-You can search your dataset via API for similar examples using the `POST /datasets/<id>/search` REST API. Its documentation
-can be found [here](https://api.smith.langchain.com/docs#/datasets/search_api_v1_datasets__dataset_id__search_post).
-You can see detailed examples of how use this in your prompts both with and without LangChain in
-[our cookbook on using indexed datasets with few shot prompts](https://github.com/langchain-ai/langsmith-cookbook/blob/beta-few-shot-search-cookbook/optimization/dataset-few-shot-search/FewShotDatasetsQuickstart.ipynb)
```

Comment:
Now that we have an in app code link, and we are going to have https://github.com/langchain-ai/langchain/pull/25601, I think the cook book is less useful. 

I could link to https://github.com/langchain-ai/langtweet/tree/main if we want as an exmaple when harrison makes it public?

Reply:
Agree, take out the cookbook. But should inline the code snippet of how to use the SDK (Python and JS when available) to search for similar examples. We have code snippets for SDK methods in various other parts of the documentation